src/Controller/UsersController.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\File\UploadedFile;
+use App\Entity\User;
+use Symfony\Component\HttpFoundation\File\Exception\FileException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\String\Slugger\SluggerInterface;
+use App\Form\UserType;
+use Symfony\Component\HttpFoundation\File\File;
+
+
+
+class UsersController extends AbstractController
+{
+    #[Route('/users', name: 'app_users')]
+    public function index(): Response 
+    {
+        return $this->render('users/index.html.twig', [
+            'controller_name' => 'UsersController',
+        ]);
+    }
+
+    #[Route('/user/files', name: 'app_user_new')]
+    public function new(Request $request, SluggerInterface $slugger): Response
```

Comment:
main request Products is changed for Users view:
https://symfony.com/doc/current/controller/upload_file.html